.ci/mellanox/azure-pipelines.yml

```diff
@@ -0,0 +1,51 @@
+# Per-commit
+trigger:
+  - master
+# All PRs
+pr:
+  branches:
+    include:
+    - '*'
+
+pool:
+  name: Default
+  demands: AGENT_CI_CAPABILITY -equals ompi
+
+variables:
+  # TODO: change to the main repo and master branch
+  #ompi_jenkins_scripts_git_repo_url: https://github.com/mellanox-hpc/jenkins_scripts.git
+  ompi_jenkins_scripts_git_repo_url: https://github.com/itemko/jenkins_scripts.git
```

Comment:
Are you sure you want the file in `itemko`, not `mellanox-hpc`?

Reply:
It will be switched to `mellanox-hpc` repo and `master` branch after PR https://github.com/mellanox-hpc/jenkins_scripts/pull/88 (we've mostly reviewed the changes internally - just waiting for the final approval from the reviewers) - right now there're forked scripts to enable the overall CI solution and verify it with the mainline Open MPI repo.